src/Sulu/Component/Rest/ListBuilder/Metadata/ListXmlLoader.php

```diff
@@ -203,6 +206,19 @@ private function loadCountPropertyMetadata(\DOMXPath $xpath, \DOMNode $propertyN
         return $propertyMetadata;
     }
 
+    private function loadCountDistinctPropertyMetadata(\DOMXPath $xpath, \DOMNode $propertyNode)
```

Comment:
Maybe not needed if we change the exist adapter. Lets wait for the response of the other core teams members.